.github/workflows/ruby.yml

```diff
@@ -51,3 +51,10 @@ jobs:
         run: bundle exec erblint --lint-all
       - name: Run i18n-tasks
         run: bundle exec i18n-tasks health
+
+  brakeman:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Analyze code statically using Brakeman
+        uses: moneyforward/brakeman-action@v0
```

Comment:
It seems this action is no longer maintained: https://github.com/moneyforward/brakeman-action. Let's pick one of the others.